compiler-core/src/language_server/tests/common/mod.rs

```diff
@@ -0,0 +1,253 @@
+use std::{
```

Comment:
Could you revert this and return this code to its original location please? We don't have a `common/mod.rs` convention.